### DISCUSSION TITLE
Información adicional al pinchar sobre los puertos de entrada/salida

### DISCUSSION THREAD

**Discussion by jospicant** [FPGAwars/icestudio#851] [Ideas]
Hola:

Se me ocurre como posible mejora a la hora de crear un módulo en Icestudio que cuando se introducen los puertos de entrada y salida, que estos pudieran registrar algún tipo de información que se puede visualizar al colocarse con el ratón encima del puerto.

por ejemplo, si creamos un modulo con dos puertos de entrada "a","b" y uno de salida "c" y les hemos añadido un puerto de entrada/salida desde  "Basic-->Input/Ouput" estaría bien que se pudiera añadir una información adicional a cada uno de los puertos de forma que pasamos con el ratón por el puerto "a,c, o c" nos abriese un ventana emergente que nos pudiera dar la descripción que hubiéramos podido añadir a ese puerto como por ejemplo "a: puerto de entrada activa a nivel alto, cuando pase a nivel bajo se cargará el dato del puerto b".

Sería un forma de poder introducir una información adicional a cada uno de los puertos que solo se visualizaría al poner el ratón encima de él.

Saludos y Gracias

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
